### ISSUE TITLE
[Improvement] remove 'release' from tgz

### ISSUE THREAD

**Issue by pouicr** [ncarlier/webhookd#13]
Hi,

I created an ansible recipe installing the webhookd by downloading the tgz.
Having the "release" folder inside the tgz make the installation more complex (3 steps instead of 1).

Ok it is not very important but since it doesn't seems to add any feature I guess, it may be interesting to remove it?

**Comment by pouicr** [comment]
Thx!!!

I confirm `release` folder is gone.  
Saddly the new `gz`archive format is no more inline to simplify the ansible recipe :cry: 

Not really important... reason here: https://github.com/ansible/ansible-modules-core/issues/3241

**Comment by ncarlier** [comment]
A good practice seems to use TGZ but with at the root: the binary, the release note, the license and the readme.

You will get something like:

```bash
tar tvzf webhookd-linux-amd64.tgz
README.md
LICENSE
CHANGELOG.md
webhookd
```
I think it's more appropriate and with such a structure, you will use Ansible again simply.

**Comment by ncarlier** [comment]
Fixed by the v1.6.1
